### PR TITLE
Original Fxml Files

### PR DESCRIPTION
**Pull request by cn20kaman**
Created four fxml files:
- [x] Patient Journal fxml
- [x] Prescription fxml
- [x] User Profile fxml
- [x] Visit fxml

### CODE REVIEW

File: src/main/resources/Journal.fxml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                  <Label layoutX="651.0" layoutY="424.0" text="State:" />
+                  <ChoiceBox layoutX="651.0" layoutY="464.0" prefWidth="150.0" />
+                  <Label layoutX="334.0" layoutY="424.0" text="Town:" />
+                  <TextField layoutX="336.0" layoutY="463.0" prefHeight="57.0" prefWidth="304.0" />
+                  <TextField layoutX="16.0" layoutY="463.0" prefHeight="57.0" prefWidth="304.0" />
+                  <Label layoutX="14.0" layoutY="424.0" text="Street:" />
+                  <Label layoutX="14.0" layoutY="384.0" text="Billing Address:" />
+                  <Label layoutX="14.0" layoutY="544.0" text="Other Doctors:" />
+                  <ListView layoutX="13.0" layoutY="584.0" prefHeight="145.0" prefWidth="314.0" />
+                  <Button layoutX="347.0" layoutY="583.0" mnemonicParsing="false" text="Add Another Doctor" />
```
Comment (LeoPeg-732) [inline src/main/resources/Journal.fxml:47]: I pretty much used Hboxes and Vboxes to organize the layout of the textboxes and labels. I don't think this entirely matters, but It still looks nice as is.

File: src/main/resources/Journal.fxml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                  <TextField layoutX="331.0" layoutY="153.0" prefHeight="57.0" prefWidth="83.0" />
+                  <Label layoutX="331.0" layoutY="114.0" text="DOB:" />
+                  <Label layoutX="431.0" layoutY="114.0" text="Primary Doctor:" />
+                  <TextField layoutX="431.0" layoutY="153.0" prefHeight="57.0" prefWidth="172.0" />
+                  <Label layoutX="14.0" layoutY="234.0" text="Mailing Address:" />
+                  <Label layoutX="14.0" layoutY="274.0" text="Street:" />
+                  <TextField layoutX="16.0" layoutY="313.0" prefHeight="57.0" prefWidth="304.0" />
+                  <TextField layoutX="336.0" layoutY="313.0" prefHeight="57.0" prefWidth="304.0" />
+                  <Label layoutX="334.0" layoutY="274.0" text="Town:" />
+                  <ChoiceBox layoutX="651.0" layoutY="314.0" prefWidth="150.0" />
```
Comment (LeoPeg-732) [inline src/main/resources/Journal.fxml:32]: I didn't use a choicebox. I used a menubutton, but I do think a choice box is a better option.

**Review by LeoPeg-732 — COMMENTED** [review]
We have different size text boxes, but other than that, I think this is fine.

**Review by LeoPeg-732 — COMMENTED** [review]
Same as the previous comment. Our sizes aren't the same for textboxes and labels. We can discuss what size is appropriate,

File: src/main/resources/Journal.fxml [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<Accordion maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="1080.0" prefWidth="1920.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (tlacasse) [inline src/main/resources/Journal.fxml:7]: We will need to specify controllers, but the controller attribute can be added later with the actual controller. just a note though.

**Pull request merged**